mmligner.py

```diff
@@ -29,166 +29,133 @@
         Filename: *
       Request: FileLocations
 """
-#!/usr/bin/python
-import yasara,string,disk,os
-import mmaligneralign
-#import mmlignerutilities,mmlignerplotoutput
 
-from python2to3 import *
+import os
+import yasara
+import disk
 
-# amino acid residue list
-aminoacidDict = {}
-aas = ['A','C','D','E','F','G','H','I','K','L','M','N','P','Q','R','S','T','V','W','Y','y','p','s','z']
+from mmligneralign import align_molecules, YObject
 
-aas2 = ['ALA','CYS','ASP','GLU','PHE','GLY','HIS','ILE','LYS','LEU','MET','ASN','PRO','GLN','ARG','SER','THR','VAL','TRP','TYR','PTR','TPO','SEP','TYS']
 
-nucs=['a','c','g','t','u']
+ERROR_MESSAGE = """Cannot execute MMligner.
+ Make sure you have set the correct MMligner file
+ locations in Analyze > Align > Configure plugin.
+"""
 
-for index in range( len(aas) ):
-  aminoacidDict[ aas2[index ] ] = aas[index]
+SUCCESS_MESSAGE = """
+Aligned PDB loaded in YASARA soup as Object {}.
+ You can save this PDB when the plugin procedure ends. Please wait ...
+"""
 
-# set non-calculation requests so we don't clean the cache folder after these. 
-noncalc = ['FileLocations']
-# redundant files that can be removed when saving FoldX output to new folder
-redundantfiles = ['PdbList_']
 
-# Main program
-# ============
+def mmligner_cache(mmligner_tmp):
+  cache = os.path.join(os.getcwd(), mmligner_tmp)
+  if not os.path.isdir(cache):
+    disk.remove(cache)
+  if not os.path.exists(cache):
+    disk.makedirs(cache, yasara.permissions)
+
+  return cache
+
+def mmligner_exe(mmligner_path):
+  from subprocess import call
+
+  call(mmligner_path)
```

Comment:
for non-obvious reasons, on my Windows machine, the subprocess does not start.

Reply:
Eek! That's not good. Can you paste an error message here? Or maybe we sit together to look at it tomorrow

Reply:
I'll try to trace down the exact line where it stops. Now in line 127.

Reply:
I pushed an extra commit that changes:
> `call(mmligner_path)`

to
> `call([mmligner_path])`

I doubt it will fix the issue, but I thought it worth a try

Reply:
this fix is not necessary, I was too fast when answering.